pet.el

```diff
@@ -118,7 +118,12 @@ and nil otherwise."
 
 (defun pet-system-bin-dir ()
   "Determine the correct script directory based on `system-type'."
-  (if (eq system-type 'windows-nt) "Scripts" "bin"))
+  (if (eq (if (file-remote-p default-directory)
+              (tramp-get-connection-property (tramp-dissect-file-name default-directory)
+                                             "uname" 'windows-nt)
```

Comment:
why is this property defaulting to windows?

Reply:
The only machines without `uname` installed are windows.